help/en/html/doc/Technical/IntroStructure.shtml

```diff
@@ -114,8 +113,7 @@
           Contains commonly useful <strong>t</strong>ools and extensions.<br>
           It can depend on jmri.* and externals. It must not depend on jmrix.* <a href=
           "images/OverallImplementationStructure.png"><img src=
```

Comment:
Image out of place.  Possibly move to line 101.

Reply:
Thanks, image was moved down